service/gcsutils/gcstools/commoncli/common.go

```diff
@@ -79,6 +80,15 @@ func SetupLogging(args ...*string) error {
 	if len(args) < 1 {
 		return fmt.Errorf("Invalid log params")
 	}
-	utils.SetLoggingOptions("verbose", *args[0])
+
+	logrus.AddHook(commonutils.NewStackHook([]logrus.Level{logrus.DebugLevel}))
```

Comment:
Interesting. I think that we want this then:
```go
if level == logrus.DebugLevel {
    logrus.AddHook(commonutils.NewStackHook(logrus.AllLevels))
}
```

So only if the user says -loglevel:debug do we log stacks but we do it for any log message